src/main/java/com/checkmarx/flow/config/FlowConfig.java

```diff
@@ -11,7 +12,7 @@
 import org.springframework.web.client.RestTemplate;
 
 import java.beans.ConstructorProperties;
-import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
```

Comment:
is this deliberate?

Reply:
Yes, this is to replace Charset.forName("UTF-8")
with StandardCharsets.UTF_8, which looks neater to me.